modelling/src/main/java/org/axonframework/modelling/command/AggregateAnnotationCommandHandler.java

```diff
@@ -472,6 +498,32 @@ public boolean canHandle(CommandMessage<?> message) {
         }
     }
 
+    private Object handleNewInstanceCreation(CommandMessage<?> command,
+                                                    CreationPolicyAggregateFactory<T> factoryMethod,
+                                                    MessageHandlingMember<? super T> handler) throws Exception {
+        AtomicReference<Object> response = new AtomicReference<>();
+        AtomicReference<Exception> exceptionDuringInit = new AtomicReference<>();
+        VersionedAggregateIdentifier commandMessageVersionedId = resolveNullableVersionedAggregateIdentifier(
```

Comment:
In case of the `AggregateCreateOrUpdateCommandHandler` invoking this method, we already know the `commandMessageVersionedId` is `null`.

Reply:
So, why not make it a parameter to this private method?

Reply:
Or, differently put, make the `commandMessageAggregateId` a parameter to this method.